src/Ryujinx/Ui/MainWindow.cs

```diff
@@ -601,7 +619,7 @@ private void InitializeSwitchInstance()
 
         private SurfaceKHR CreateVulkanSurface(Instance instance, Vk vk)
         {
-            return new SurfaceKHR((ulong)((VKRenderer)RendererWidget).CreateWindowSurface(instance.Handle));
+            return new SurfaceKHR((ulong)((VkRenderer)RendererWidget).CreateWindowSurface(instance.Handle));
```

Comment:
```suggestion
            return new SurfaceKHR((ulong)((VKRenderer)RendererWidget).CreateWindowSurface(instance.Handle));
```

Or maybe rename the original class `VulkanRenderer` ?